Air pollution prediction/plotting.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+from matplotlib import pyplot as plt 
+
+
+fig,ax = plt.subplots(figsize=(12,8))
+
+plot_start = 225
+# plot the training data 
+ax.plot(y_train.index[plot_start:],y_train.values[plot_start:],'navy',marker='o',label='observed')
```

Comment:
Same thing over here and in line `11` ,`12`,`14`,`15`,`16` ,  you haven't defined Y_train , pred value anywhere

Reply:
She has split the code into several files without any link to each other. @Pranjal-2001 You need to structure your code as done in PR #556 or #638